src/main/io/serial.c

```diff
@@ -350,7 +350,7 @@ bool isSerialConfigValid(serialConfig_t *serialConfigToCheck)
             // Ensure that the baud rate on soft serial ports is limited to 19200
             serialConfigToCheck->portConfigs[index].gps_baudrateIndex = constrain(portConfig->gps_baudrateIndex, BAUD_AUTO, BAUD_19200);
             serialConfigToCheck->portConfigs[index].blackbox_baudrateIndex = constrain(portConfig->blackbox_baudrateIndex, BAUD_AUTO, BAUD_19200);
-            serialConfigToCheck->portConfigs[index].telemetry_baudrateIndex = constrain(portConfig->telemetry_baudrateIndex, BAUD_AUTO, BAUD_19200);
+            // serialConfigToCheck->portConfigs[index].telemetry_baudrateIndex = constrain(portConfig->telemetry_baudrateIndex, BAUD_AUTO, BAUD_57600);
```

Comment:
This is still valid but was being overridden by the other change below.

Reply:
With this line in place it does not work